packages/web/components/bridge/deposit-address-screen.tsx

```
@@ -0,0 +1,567 @@
+
+    const { hasCopied, onCopy } = useClipboard(
+      data?.depositData?.depositAddress ?? "",
+      3000
+    );
+
+    const expirationTimeDayjs = data?.depositData?.expirationTimeMs
+      ? dayjs(data.depositData.expirationTimeMs)
+      : undefined;
+
+    const isExpired =
+      expirationTimeDayjs && expirationTimeDayjs.isBefore(dayjs());
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)